controllers/user.controller.js

```diff
@@ -9,6 +10,40 @@ const {
   getStartDayOfKST,
 } = require("../utils/helpers");
 
+const TIER_LIST = [
```

Comment:
Please consider making global constant variable TIER_LIST, so you can use it at data and user controllers